src/script/ceph-release-notes

```diff
@@ -195,6 +195,7 @@ def make_release_notes(gh, repo, ref, plaintext, verbose, strict, use_tags):
             if use_tags:
                 title = split_component(title, gh, number)
 
+            title = re.sub(r'\.\s*$', '', title) # strip trailing punctuation
```

Comment:
wouldn't a normal title.strip() suffice here? (not sure if I'm missing anything though)